src/details/fields/abstract-field-template.ts

```diff
@@ -102,4 +108,26 @@ export abstract class AbstractFieldTemplate implements FieldTemplate {
         }
         return fieldValue;
     }
+
+    protected async generateAttachmentContent(html: string): Promise<string> {
+        let returnHtml: string = html;
+        let matchAllImage = html.match(/<img [^>]*src="([^"]+)"[^>]*>/g);
+        if (matchAllImage) {
+            for (let image of matchAllImage) {
+                if (image) {
+                    let matchImage = image.match(/<img [^>]*src="([^"]+)"[^>]*>/);
```

Comment:
Please make sure we always return a value, even if embedded image replacement errors out.